picard/coverart/providers/caa_release_group.py

```diff
@@ -38,17 +38,17 @@ class CoverArtProviderCaaReleaseGroup(CoverArtProviderCaa):
     """Use cover art from album release group"""
 
     NAME = "CaaReleaseGroup"
+    TITLE = N_(u"CAA Release Group")
```

Comment:
This should be "Cover Art Archive Release Group" because the CAA provider itself uses "Cover Art Archive".